packages/ckb-indexer/tests/transaction_collector.test.ts

```diff
@@ -190,7 +190,7 @@ test("input cell can be found in transaction detail", async (t) => {
     .returns(ioTypeInputResult);
   const cellCollector = new TransactionCollector(indexer, queryOption, nodeUri);
   const count = await cellCollector.count();
-  t.is(count, 2);
+  t.is(count, 1);
```

Comment:
change 2 to 1 cause I delete some test data, 
now the test flow change to I have only one transaction that ioType is input.and finally I count the result, there is also 1.
maybe I can change 1 to a variable named expectedCount

Reply:
why we should DELETE test data?